addons/website_livechat/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -0,0 +1,31 @@
+odoo.define('website_livechat/static/src/models/messaging_notification_handler/messaging_notification_handler.js', function (require) {
+'use strict';
+
+const { registerInstancePatchModel } = require('mail/static/src/model/model_core.js');
+
+registerInstancePatchModel('mail.messaging_notification_handler', 'website_livechat/static/src/models/messaging_notification_handler/messaging_notification_handler.js', {
+
+    //----------------------------------------------------------------------
+    // Private
+    //----------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    _handleNotificationPartner(data) {
+        const { info } = data;
+        if (info === 'send_chat_request') {
+            this._handleNotificationPartnerChannel(data);
```

Comment:
Isn't `_super` already taking care of doing this?
Feels like the only extra here is to do here is to open the thread actively.

Reply:
I prefer pre-emptive return here, not rely on `_super`. Especially considering it's a side-effect of improper fallback and all...

Reply:
But there's no preemptive return, it's doing the job twice here.

Reply:
Just force-pushed with added "pre-emptive" return (more a break, meaning notification is being fully handled in this conditional branch)

Reply:
ok